src/stories/pagefold/Pagefold.tsx

```diff
@@ -5,7 +5,7 @@ export type PagefoldProps = {
   isInheriting: boolean;
   isAContainer: boolean;
   size: "none" | "xsmall" | "small" | "medium" | "large" | "xlarge";
-  type?: "success" | "alert";
+  type?: string;
```

Comment:
why not being more strict here and specify known types?

Reply:
I fixed this to work in a more declarative way :)